pkg/system/time_linux.go

```diff
@@ -0,0 +1,42 @@
+package system
+
+import (
+	"sync"
+	"time"
+
+	"github.com/tklauser/go-sysconf"
+	"golang.org/x/sys/unix"
+)
+
+var sysconfClockTickOnce, bootTimeOnce sync.Once
+
+// The default clock tick in sysconf is 100. Never use this constant directly and only access the
+// value via `getClockTicks`, as the user might change this value.
```

Comment:
so why have this value here then?

Reply:
It acts as a fallback. If you look at the `GetClockTicks` code, if we encounter an error when loading the sysconf value, we can fallback to `100`. Anyway this value will be `100` for pretty much all systems out there, besides maybe IoT devices.